test/test_jit_cuda_fuser.py

```diff
@@ -4393,6 +4393,27 @@ def f(x):
         self.assertEqual(prof.events().table().find("fallback"), -1)
         torch._C._jit_set_nvfuser_guard_mode(old_guard)
 
+    @unittest.skipIf(not RUN_NVFUSER, "requires CUDA")
+    @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.PROFILING,
+                     "Requires fusion optimization pass to be effective")
+    @unittest.skipIf(is_pre_volta(), "reduction not supported in pre volta device")
+    def test_inf_quick_patch(self):
+        x = torch.tensor([-float('inf'), -float('inf'), 4.0], device="cuda")
```

Comment:
This test is not enough to catch previously problematic values - you need input full of -inf (nothing else there) to catch that FLT_MIN is not the correct initializer, similarly for amin. You need yet another input for nan propagation (unless you have those tests in other places, in which case it's probably better to unify)